src/java.base/share/classes/javax/crypto/EncryptedPrivateKeyInfo.java

```diff
@@ -73,19 +77,14 @@ public class EncryptedPrivateKeyInfo {
     private final byte[] encoded;
 
     /**
-     * Constructs (i.e., parses) an {@code EncryptedPrivateKeyInfo} from
-     * its ASN.1 encoding.
-     * @param encoded the ASN.1 encoding of this object. The contents of
-     * the array are copied to protect against subsequent modification.
-     * @exception NullPointerException if the {@code encoded} is
-     * {@code null}.
-     * @exception IOException if error occurs when parsing the ASN.1 encoding.
+     * Constructs an {@code EncryptedPrivateKeyInfo} from a given Encrypted
+     * PKCS#8 ASN.1 encoding.
+     * @param encoded the ASN.1 encoding to be parsed.
+     * @throws NullPointerException if {@code encoded} is {@code null}.
+     * @throws IOException if error occurs when parsing the ASN.1 encoding.
```

Comment:
Why change the old spec? There seems to be no problem. Especially, why remove the "array are copied" sentence?

Reply:
I can re-add the cloning.  As far as the others, `parses` was implied by `from an ASN.1 encoding.` and I thought being more specific about what type of ASN.1 encoding was better